.github/workflows/conformance-k8s-kind.yaml

```diff
@@ -0,0 +1,181 @@
+name: ConformanceK8sKind
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  kind_version: v0.17.0
+  kind_config: .github/kind-config-noproxy.yaml
+  cluster_name: cilium-testing
+  # renovate: datasource=github-releases depName=cilium/cilium-cli
+  cilium_cli_version: v0.13.0
+  e2e_k8s_version: v1.26.0
```

Comment:
just a question, can you share the repo for this, so that we can configure renovate bot to upgrade for the new k8s release?

Reply:
can you elaborate?

Reply:
Ideally we automatically pick up k8s patch releases via renovate. "Minor" releases would still need to be automatic.

Given that we bump k8s versions manually on other places, it's not critical.

Reply:
which repo is the conformance test residing ? Is it part of k8s main repo ? Basically, I would love to configure datasource for renovate bot to update if there is new release?

Reply:
this is stored on kubernetes release buckets, it is created every time a new release is cut

But I changed it to match the kind node images, so the e2e and the kubernetes version match, there is no guarantee that e2e tests work against different versions, but is not common we break that though